### PR TITLE
Improve durable consumer unsubscribe comment

### PR DESCRIPTION
**Pull request by sylr**
Signed-off-by: Sylvain Rabot <sylvain@abstraction.fr>

### CODE REVIEW

File: js.go [hunk 1]
```diff
@@ -67,9 +67,11 @@ type JetStream interface {
 	// gone.
 	// * If Durable() option is specified, the library will attempt to lookup a JetStream
 	// consumer with this name, and if found, will bind to it and not attempt to
-	// delete it. However, if not found, the library will send a request to create
-	// such durable JetStream consumer. The library will delete the JetStream consumer
-	// after an Unsubscribe() or Drain().
+	// delete it. However, if not found, the library will send a request to
+	// create such durable JetStream consumer. Note that the library will delete
+	// the JetStream consumer after an Unsubscribe() or Drain() *only* if it
+	// created the durable consumer while subscribing. if the durable consumer
```
Comment (piotrpio) [inline js.go:73]:
```suggestion
	// created the durable consumer while subscribing. If the durable consumer
```

File: js.go [hunk 2]
```diff
@@ -67,9 +67,11 @@ type JetStream interface {
 	// gone.
 	// * If Durable() option is specified, the library will attempt to lookup a JetStream
 	// consumer with this name, and if found, will bind to it and not attempt to
-	// delete it. However, if not found, the library will send a request to create
-	// such durable JetStream consumer. The library will delete the JetStream consumer
-	// after an Unsubscribe() or Drain().
+	// delete it. However, if not found, the library will send a request to
+	// create such durable JetStream consumer. Note that the library will delete
+	// the JetStream consumer after an Unsubscribe() or Drain() *only* if it
```
Comment (piotrpio) [inline js.go:72]: Not sure about additional formatting in the comment, the formatting will not be used or visible in godoc.
```suggestion
	// the JetStream consumer after an Unsubscribe() or Drain() only if it
```

**Review by piotrpio — CHANGES_REQUESTED** [review]

**Review by kozlovic — CHANGES_REQUESTED** [review]
I agree with @piotrpio recommendations

**Review by kozlovic — APPROVED** [review]
LGTM

**Review by piotrpio — APPROVED** [review]
LGTM

**Pull request merged**